Bio/Medline/__init__.py

```diff
@@ -166,8 +166,15 @@ def parse(handle):
     key = ""
     record = Record()
     for line in handle:
+        orig_line = line
```

Comment:
I think there is an easier way to handle this, without creating `orig_line`, by postponing the call to `rstrip` until after checking for `line[:6] == "      "`.